packages/http/src/utils/__tests__/parseResponse.spec.ts

```diff
@@ -78,7 +78,7 @@ describe('parseResponseHeaders()', () => {
     expect(parseResponseHeaders({ h1: ['a b'], h2: ['c'], h3: ['a', 'b'] })).toEqual({
       h1: 'a b',
       h2: 'c',
-      h3: 'a b',
+      h3: 'a,b',
```

Comment:
We'll also have to write a more specific test to make sure it only happens only under specific conditions.